.github/workflows/linux.yml

```diff
@@ -45,7 +45,7 @@ env:
   RUST_BACKTRACE: 1
   SHELL: /bin/bash
   SCCACHE_GHA_ENABLED: "true"
-  RUSTC_WRAPPER: "sccache"
+  #RUSTC_WRAPPER: "sccache"
```

Comment:
Perhaps this was left by mistake?

Reply:
No, sccache issue is still not resolved, but I have [a fix PR](https://github.com/mozilla/sccache/pull/1897) maybe we can use it.

Reply:
I would remove this line then, rather than leaving it commented out.